switch_mod/utilities.py

```diff
@@ -223,11 +221,9 @@ def post_solve(model, outputs_dir=None):
     # (the latter occurs when there are problems with licenses, etc)
     
     # replace the old _save_results function
```

Comment:
What do you think about the TODO comment I left? Should that be left there or should I initiate a new GitHub Issue thread regarding that topic and erase the comment lines in the code?